### PR TITLE
implemented Invoice Screen feature

### PR DESCRIPTION
**Pull request by TejasDhole**
**Changes in existing code base:**

1.added color used in invoice screen 
2.added invoice option inside dashboard

**Created new feature:**
1.Created Invoice screen feature 
2.Created Add Invoice Dialog box

### CODE REVIEW

File: lib/constants/color.dart [hunk 1]
```diff
@@ -38,6 +38,9 @@ const Color greyColor = Color(0xff9E9E9E);
 const Color borderblue = Color.fromRGBO(0, 0, 255, 0.27);
 const Color lightBlueColor = Color(0xff0077B6);
 const Color darkOrangeColor = Color(0xFFE75347);
+const Color searchBar = Color.fromARGB(255, 143, 143, 143);
```
Comment (Anik-Chakraborty) [inline lib/constants/color.dart:41]: searchBar is already  defined.. it's generating error... fix this

Comment (Anik-Chakraborty) [reply lib/constants/color.dart:41]: remove this

Comment (TejasDhole) [reply lib/constants/color.dart]: removed

**Comment by Anik-Chakraborty** [comment]
image
elements are not properly aligned and test in header is also not centered.

**Comment by Anik-Chakraborty** [comment]
image
Upload and Create Invoice button is not working...

**Comment by TejasDhole** [comment]
> image Upload and Create Invoice button is not working...

still working on upload and create invoice

**Comment by Anik-Chakraborty** [comment]
Create Invoice btn doesn't refresh the table and also doesn't close the drawer

**Comment by TejasDhole** [comment]
> Create Invoice btn doesn't refresh the table and also doesn't close the drawer

you can test it ,fixes all the issues

**Pull request merged**